hypervisor/dm/vpci/partition_mode.c

```diff
@@ -54,39 +55,41 @@ static inline bool is_valid_bar(const struct pci_bar *bar)
 	return (is_valid_bar_type(bar) && is_valid_bar_size(bar));
 }
 
+/**
+ * @pre vdev != NULL
+ */
 static void partition_mode_pdev_init(struct pci_vdev *vdev, union pci_bdf pbdf)
 {
 	struct pci_pdev *pdev;
 	uint32_t idx;
 	struct pci_bar *pbar, *vbar;
 
 	pdev = find_pci_pdev(pbdf);
-	if (pdev != NULL) {
-		vdev->pdev = pdev;
-
-		/* Sanity checking for vbar */
-		for (idx = 0U; idx < (uint32_t)PCI_BAR_COUNT; idx++) {
-			pbar = &vdev->pdev->bar[idx];
-			vbar = &vdev->bar[idx];
-
-			if (is_valid_bar(pbar)) {
-				vbar->size = (pbar->size < 0x1000U) ? 0x1000U : pbar->size;
-				vbar->type = PCIBAR_MEM32;
-			} else {
-				/* Mark this vbar as invalid */
-				vbar->size = 0UL;
-				vbar->type = PCIBAR_NONE;
-			}
+	ASSERT(pdev != NULL, "pdev is NULL");
+
+	vdev->pdev = pdev;
```

Comment:
Is this SPACE or TAB? I saw the visual display is different with others.

Reply:
TAB